pilot/docker/Dockerfile.proxy

```diff
@@ -1,5 +1,9 @@
-# keep Dockerfile.proxy and Dockerfile.proxy_debug consistent for proxy sha
-FROM gcr.io/istio-testing/envoy:54077a0c8486b6db7b2fc5d10619b6813bb40767
+FROM ubuntu:xenial
+
+# Install and configure Envoy.
```

Comment:
The envoy image also installed iptables and added:

<pre>ADD istio-iptables.sh /usr/local/bin/istio-iptables.sh
ADD istio-start.sh /usr/local/bin/istio-start.sh</pre>

are these not needed?

Reply:
I don't believe so ... those were the old iptables scripts. The current iptables are set up by the init container and uses the script from istio/istio.

Reply:
Do we really something as big as xenial ?

Reply:
Just being consistent with what the image was previously. Changing it would seem beyond the scope of this PR.

Reply:
Please add the iptables and istio-start - the docker image can also be used as a basis and outside k8s.

It would be great if this PR copied exactly the dockerfile used in proxy - and we can have a 
separate PR if we want to make changes.